packages/lit-html/src/directives/cache.ts

```diff
@@ -21,13 +22,25 @@ import {
   clearPart,
   getCommittedValue,
   insertPart,
+  isCompiledTemplateResult,
   isTemplateResult,
   setCommittedValue,
 } from '../directive-helpers.js';
 
+/**
+ * @returns The template strings array for the provided template.
+ */
+function getStringsFromTemplate(
+  template: TemplateResult | CompiledTemplateResult
```

Comment:
It's not a template, but a template result. I would name it `result`, and put "result" in the function name too.

Reply:
I.e. `getStringsFromTemplateResult`?